krun/platform.py

```diff
@@ -937,6 +942,13 @@ def _find_virt_what(self):
             fatal("virt-what is not installed")
         return exe
 
+    def clear_cpu_pinning(self):
+         if os.path.exists(LinuxPlatform.USER_CSET_DIR):
```

Comment:
This only exists if a shield is in place?

Reply:
We should probably find out to avoid nasty surprises. Can you look into it?

Reply:
I didn't really realize it stored the info of active shields. What i really wanted is does the cpuset binary exist. Also how do i tell if the cpuset was created by us?

Reply:
I think a safer bet would be to only do the pinning bits if pinning is enabled in the user's config. We already check for the userspace tools:

https://github.com/softdevteam/krun/blob/2ef1e77703d0356dccefb84132f7643bc223e021/krun/platform.py#L912-L916

You can use `if self.config.ENABLE_PINNING:` just like we have done above.

Reply:
i just changed it to if self.config.ENABLE_PINNING